apps-rendering/src/components/list.tsx

```diff
@@ -23,14 +29,28 @@ const styles: SerializedStyles = css`
 		background-color: ${neutral[86]};
 		margin-left: -${remSpace[6]};
 		vertical-align: middle;
+
+		${darkModeCss`
+		${
+			isStandfirst && format
+				? `background-color: ${background.liveblogStandfirstBulletDark(
+						format,
+				  )};`
+				: `background-color: ${neutral[46]};`
+		}
+		`}
 	}
 `;
 
 interface Props {
 	children: ReactNode;
+	isStandfirst?: boolean;
+	format?: ArticleFormat;
```

Comment:
Can we make these non-optional? I think it's good to be explicit with function/component arguments, and we have `format` and `isStandfirst` available in the other place this is used in `renderer`. I also think it makes the condition above slightly simpler because you don't have to include the check for the existence of `format`.